views/layouts/main.hbs

```diff
@@ -119,7 +123,7 @@
       Leaderboard
     </a>
 
-    <div class="ui simple dropdown item">
+    <div class="ui simple dropdown item {{menu.claims}}">
```

Comment:
`claims-active` isn't a valid class. it should be `active` only

Reply:
Adding active class along with drop-down would show drop-down box. So i added claim-active class and defined it in main.hbs